open_event/views/admin/models_views/event.py

```diff
@@ -45,12 +45,6 @@ class EventView(ModelView):
                    'slogan',
                    'url')
 
-    column_formatters = {
-        'name': Formatter.column_formatter,
-        'location_name': Formatter.column_formatter,
-        'logo': Formatter.column_formatter
```

Comment:
I felt this was unnecessary. And the formatter was causing the `logo` URL to be truncated.